driver/CMakeLists.txt

```
@@ -18,10 +18,9 @@ if(LPAC_DYNAMIC_DRIVERS)
 else()
     add_library(euicc-drivers STATIC ${DIR_INTERFACE_SRCS})
 endif()
-target_link_libraries(euicc-drivers euicc cjson-static)
+target_link_libraries(euicc-drivers euicc cjson-static lpac_utils)
```

```suggestion
target_link_libraries(euicc-drivers euicc cjson-static lpac-utils)
```

i think this style needs to be consistent